dataScrape/apiPull.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+require_once 'src/apiPull.php';
+require_once 'src/pokemonTableUpdate.php';
```

Comment:
Could replace these two with an autoloader.

Reply:
This part of the application is standalone, so does not need to be autoloaded.